data/trek/loot_tables/fab_external.json

```diff
@@ -6,10 +6,15 @@
 			"conditions": [
 				{
 					"condition": "random_chance",
-					"chance": 0.005
+					"chance": 0.0025
 				}
 			],
 			"entries": [
+				{
+					"type": "loot_table",
+					"name": "minecraft:gameplay/fishing/junk",
```

Comment:
Might be interesting to change this to `minecraft:gameplay/hero_of_the_village/shepherd_gift` and also give out shears so people can modify the ship in that way